projects/packages/forms/tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
 	"extends": "jetpack-js-tools/tsconfig.base.json",
 	// List all sources and source-containing subdirs.
-	"include": [ "./src", "./declarations.d.ts" ],
-	"files": [ "./global.d.ts" ]
+	"include": [ "src", "declarations.d.ts", "global.d.ts" ]
```

Comment:
This is to avoid out-of-scope type checking. For example, without this change, it can type-check the consumed packages like `components`.